scss/components/_case-card.scss

```diff
@@ -165,6 +165,13 @@
     color: $gray-dark;
   }
 
+  .civicase__case-card-title {
```

Comment:
This is not the title that we show on the UI? we show the subject, class name should be the same.